src/python/pants/backend/jvm/tasks/reports/junit_html_report.py

```diff
@@ -112,7 +112,7 @@ def __lt__(self, other):
   def success_rate(test_count, error_count, failure_count, skipped_count):
     if test_count:
       unsuccessful_count = error_count + failure_count + skipped_count
-      return '{:.2f}%'.format((test_count - unsuccessful_count) * 100.0 / test_count)
+      return f'{((test_count - unsuccessful_count) * 100.0 / test_count):.2f}%'
```

Comment:
I think the outermost parentheses are unnecessary..would be good to confirm via REPL.